src/server.js

```diff
@@ -373,6 +373,19 @@ function initializeClient(socket, client, token, lastMessage) {
 		client.unregisterPushSubscription(token);
 	});
 
+	socket.on("logs:download", (data) => {
+		const network = "irc.freenode.org"; // FIXME: Retrieve this from client one way or another
+		const logFolder = Helper.getUserLogsPath(client.name, network);
```

Comment:
@xPaw, do you have any suggestion for passing network + channel/query name from the client to the server?
I believe the client doesn't know the URL of the network, just the display name (at least for now), and I need it to browse the correct folder.
I'm thinking that instead of passing string of network+channel, I could instead pass the ID of the channel that the server knows about. Is that possible? Any suggestion?

Reply:
Making the host available is fairly easy as it is already send to the client with the "init"  event as can be seen here https://github.com/thelounge/lounge/blob/36be7565a700ab43c3f7e8ef4844d7886f88f498/src/server.js#L431 

So all it takes is to add `data-host="{{host}}"` somewhere in `network.tpl` and it is available. 

The same data is available server side so you could use the id as that simply sees to be an array key+1.

Reply:
I'd wait for https://github.com/thelounge/lounge/pull/1839/commits/d085b0692d4b047fb4ee7ffae005e4ad20c86df3 until making logs downloadable to be honest.

Reply:
> I'd wait for d085b06 until making logs downloadable to be honest.

I was thinking about that, but also I'd like to not depend on something else to be able to make progress on this. I don't mind reworking this PR once d085b06 makes it in.
I'm also thinking of just cherry-picking it for the time being, but I feel like this PR might be ready before #1839.

Reply:
It turns out, this was muuuch easier than I thought, and didn't require any trickery. As long as we're passing the channel id to the server, we can `find` whatever we need.